web/src/state/index.ts

```diff
@@ -14,3 +14,18 @@ export type Workspace = {
 };
 const workspace = atom<Workspace | undefined>(undefined);
 export const useWorkspace = () => useAtom(workspace);
+
+export type Project = {
+  id: string;
+  name: string;
+};
+
+const project = atom<Project | undefined>(undefined);
+export const useProject = () => useAtom(project);
+
+const unselectProject = atom(null, (_get, set) => {
+  set(project, undefined);
+  set(workspace, undefined);
+});
```

Comment:
This feels a little unexpected to be placed here. It also does more than its name. It says unselect project, but it also unselects workspace, right?
And where it is used, you already import useProject and useWorkspace, so I think its better, and clearer whats happening, to just move this logic into that hooks file.

Reply:
It is related to the jotai state atom which all resides in one file, same as done in re:earth (All state-related code in one file)